compare_mt.py

```diff
@@ -17,7 +18,8 @@ def parse_profile(profile):
   return kargs
 
 def print_score_report(ref, out1, out2,
-                       score_type='bleu'):
+                       score_type='bleu',
+                       sign_test=False):
```

Comment:
Instead of `sign_test=False`, maybe we could have it be `bootstrap=0`, where if you set `bootstrap` to be more than zero, it does significance testing with that many samples?